Mention deleting old map data in SQL guide (#67)
BlueMap-Minecraft/BlueMapWiki: wiki/getting-started/Installation.md

  for more info about this config-file)*
- Now open each of your map-config-files and set `storage` to `"sql"`.
- Reload BlueMap with `/bluemap reload`.
- Now that your maps are stored on your database, you should delete the old map data from your filesystem, to prevent it from interfering.  
  You can do this by running the command `/bluemap storages file delete <map>` for each of the maps you have just switched to SQL.

> Instead of running that command, you can also this manually by deleting the old map folder on your filesystem, which usually is at `bluemap/web/maps/`.

Reviewer: ```suggestion
  Or just manually delete the old map-data folder on your filesystem, which usually is at `./bluemap/web/maps`.
```
